src/Microsoft.ML.FastTree/doc.xml

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<docs>
+  <members>
+
+    <member name="FastTree">
+      <summary>
+        Trains gradient boosted decision trees to the LambdaRank quasi-gradient. 
+      </summary>
+      <remarks>
+        <para>
+          FastTree is an efficient implementation of the <a href='https://arxiv.org/abs/1505.01866'>MART</a> gradient boosting algorithm.
```

Comment:
>MART [](start = 100, length = 4)

This is a bit confusing, We say MART but the paper is an improved version DART. @tfinley@gmail.com do you know what is the correct one?

Reply:
If we use the acronym "DART" I think we need to have a link to that paper, and also explain what is meant by that. I think it's fine to just say MART honestly. The DART capabilities could perhaps be mentioned.